views/jira-configuration-new.hbs

```diff
@@ -258,6 +258,7 @@
                                 csrfToken=csrfToken
                                 successfulConnections=app.successfulConnections
                                 failedConnections=app.failedConnections
+                                appId=app.id
```

Comment:
I've done the same thing in this [PR](https://github.com/atlassian/github-for-jira/pull/1509/files#diff-a45f5fee0253b04f9afe058dfeb3d50cf5b6466ffe42276c837ec061e0902b8b), but with a different name. So, just wanted to confirm on the name, should we go with `appId` or `gitHubAppId`?

Reply:
I'll change it to gitHubAppId

Reply:
Might be worth you refactoring the `app` value elsewhere in this view at some point as well.